src/Controls/tests/Xaml.UnitTests/LoaderTests.cs

```diff
@@ -792,7 +792,7 @@ public void StyleWithoutTargetTypeThrows()
 		public void BindingIsResolvedAsBindingExtension()
 		// https://github.com/xamarin/Microsoft.Maui.Controls/issues/3606#issuecomment-422377338
 		{
-			var bindingType = XamlParser.GetElementType(new XmlType("http://schemas.microsoft.com/dotnet/2021/maui", "Binding", null), null, null, out var ex);
+			var bindingType = XamlParser.GetElementType(new XmlType("http://schemas.microsoft.com/dotnet/2021/maui", "Binding", null), null, null, true, out var ex);
```

Comment:
Is this single line change sufficient tests for this PR? Also, that link goes nowhere and seems to make some sort of indication that this test is specific.

Is there a need to check cases where a Type and TypeExtension both exist are OK?

Reply:
Binding and BindingExtension both exists